app/services/i18n/i18n.service.ts

```diff
@@ -0,0 +1,161 @@
+import electron from 'electron';
+import path from 'path';
+import VueI18n from 'vue-i18n';
+import { PersistentStatefulService } from '../persistent-stateful-service';
+import { mutation } from 'services/stateful-service';
+import recursive from 'recursive-readdir';
+import { Inject } from '../../util/injector';
+import { FileManagerService } from 'services/file-manager';
+import { IListInput, TFormData } from 'components/shared/forms/Input';
+import { I18nServiceApi } from './i18n.service-api';
+
+
+interface II18nState {
+  locale: string;
+}
+
+const I18N_PATH = path.resolve('app/i18n');
```

Comment:
The `app` directory is not included when we ship our app.  I think we should move i18n files out of `app` and into an `i18n` folder in the root of the repo.  Then I can manually include it in our package.

Reply:
hmm, I added `i18n` in package.json but didn't test it yet

Reply:
Yeah I saw you added `i18n` to extra files, but that is relative to the root of the repo, not the `app` directory.  So I think if you just move `<root>/app/i18n` to `<root>/i18n` everything will work.